src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_PER_DAY = 40;
+  const TOTAL = days * RENT_PER_DAY;
+  const THREE_DAYS_DISCOUNT = 20;
+  const SEVEN_DAYS_DISCOUNT = 50;
+  const SEVEN_DAYS = 7;
+  const THREE_DAYS = 3;
+
+  if (days < THREE_DAYS) {
+    return days * RENT_PER_DAY;
+  }
+
+  if (days >= THREE_DAYS && days < SEVEN_DAYS) {
```

Comment:
The check for 'days >= THREE_DAYS' is redundant because if the code execution reaches this point, it implies that 'days' is already at least 'THREE_DAYS' due to the return statement at line 15.